File: Python/003-number_pairs.py
# In this kata the aim is to compare each pair of integers from 
# two arrays, and return a new array of large numbers.
# Note: Both arrays have the same dimensions.

def get_larger_numbers(a: list, b: list) -> list:
    new_arr: list = []
    for i in range(len(a)):
        new_arr.append(max(a[i], b[i]))
    return new_arr


if __name__ == '__main__':
    arr1: list = [13, 64, 15, 17, 88]
    arr2: list = [23, 14, 53, 17, 80]
    print(get_larger_numbers(arr1, arr2)) # [23, 64, 53, 17, 88]

File: JavaScript/003-number_pairs.js
// In this kata the aim is to compare each pair of integers from 
// two arrays, and return a new array of large numbers.
// Note: Both arrays have the same dimensions.
'use strict';

function getLargerNumbers(a, b) {
    const newArray = [];
    for(let i = 0; i < a.length; i++) {
        newArray[i] = Math.max(a[i], b[i]);
    }

    return newArray;
}


let arr1 = [13, 64, 15, 17, 88];
let arr2 = [23, 14, 53, 17, 80];
console.log(getLargerNumbers(arr1, arr2)); // Returns [23, 64, 53, 17, 88]
